tests/e2e_tests/inference_endpoints/test_inference_endpoints.py

```diff
@@ -223,7 +224,7 @@ def test_prediction_with_certificates():
 
 
 def test_jpeg_prediction_with_certificates():
-    time.sleep(10)
+    time.sleep(30)
```

Comment:
I think that sleep is unnecessary, because one test before this, uses the same endpoint without any changes in meantime.